CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.3)
-set(QT_MIN_VERSION "5.4.0")
+
+#Set  C++14 and Qt minimum for qOveload MACRO
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(QT_MIN_VERSION "5.7.0")
```

Comment:
Why are these changes needed?

Reply:
I used qOverload macro which needed these changes. Anyway, if you intend to modernize the signal-slot syntax these changes will be useful since many signals might be overloaded.

Should I revert them?

Reply:
I want to keep compatible with current Debian stable. Does it offer a sufficiently new Qt?

If that is the case and you *do* use the `qOverload` macro, it is OK to increase the requirements (I thought I checked and did not find an instance of `qOverload` in the patch). But please modify the existing `QT_MIN_VERSION` line.

Reply:
As seen [here](https://packages.debian.org/buster/qt5-default) Debian stable is using 5.11.3. Plus, I am looking forward to make a new PR to replace the old, string-based `connect` syntax with the [new](https://doc.qt.io/qt-5/signalsandslots-syntaxes.html) one. These versions I used above are the absolute minimum for that to happen.

Of course, if you don't want that to happen, it's ok. Also, it's ok if you want the QT_MIN_VERSION to change later, when (and if) I will actually replace the connect's.

Again, I am not trying to intrude here, just doing my hobby and helping a project :)

Reply:
I've thought a bit about the case. Let's postpone the changes to `connect`s. I do not foresee that you will have to introduce more connections anyway. Please do not increase required versions only to be able to use the modern `connect` overloads.